GTMAppAuth/Sources/Public/GTMAppAuth/GTMAppAuthFetcherAuthorization+Keychain.h

```diff
@@ -33,12 +33,33 @@ NS_ASSUME_NONNULL_BEGIN
 + (nullable GTMAppAuthFetcherAuthorization *)
```

Comment:
We can add a comment in this file and `GTMKeychain.h` to remind our clients not to use methods with and without data protection altogether.

Reply:
Added guidance to method docs.